frontend/src/components/AddIcon.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useState } from "react";
+import AddBoxOutlinedIcon from "@mui/icons-material/AddBoxOutlined";
+import PostModal from "./PostModal";
+
+export default function AddIcon() {
+  const [isClick, setIsShown] = useState(false);
+
+  function handleClick() {
+    setIsShown(true);
+    console.log("clicked");
```

Comment:
You can remove this console log, and when you have time, you can style the add icon to look similar to instagram